config/deploy.rb

```diff
@@ -0,0 +1,50 @@
+# config valid only for current version of Capistrano
+lock '3.4.0'
+
+set :application, 'MythicApps-onRails'
+set :repo_url, 'https://github.com/MythicApps/MythicApps-onRails'
+
+# Default branch is :master
+# ask :branch, `git rev-parse --abbrev-ref HEAD`.chomp
+set :branch, 'atomaka/bugfix/duplicate-gems'
+
+# Default deploy_to directory is /var/www/my_app_name
+set :deploy_to, '/home/app/mythicapps'
+
+# Default value for :scm is :git
+# set :scm, :git
+
+# Default value for :format is :pretty
+# set :format, :pretty
+
+# Default value for :log_level is :debug
+# set :log_level, :debug
+
+# Default value for :pty is false
+# set :pty, true
+
+# Default value for :linked_files is []
+set :linked_files, fetch(:linked_files, []).push('.env')
```

Comment:
.env file will have to be manually placed at `/home/app/mythicapps/shared/config` (or something around there; it's late and I didn't double check the path) between `bundle exec cap production setup` and `bundle exec cap production deploy`